index.js

```diff
@@ -1 +1 @@
-module.exports = require('./lib');
\ No newline at end of file
+module.exports = require('./src/index.jsx');
```

Comment:
index.js should not import ./src/index.jsx.

npm package will have jsx to js transformed files in lib folder. which should be imported